.github/workflows/console-web-release.yml

```
@@ -38,6 +38,8 @@ jobs:
       app: console-web
       appVersion: ${{ needs.build-beta.outputs.base_image_tag }}
       environment: staging
+      serviceEnvVars: |
```

actionlint (1.7.4)

context "secrets" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)